install/crds.yaml

```diff
@@ -1084,12 +1084,12 @@ spec:
                     type: boolean
                 type: object
               grpcPort:
```

Comment:
Is this still just a port? Do we need to update the CRD here for something like `grpcAddr` and `httpAddr`

Reply:
If we change the fields from port to `grpcAddr` and http-port to `httpAddr`, we need to have some kind of migration to actually migrate the old CRs to these new values. This is why to keep it simple I have just changed the description and not fields.